### ISSUE TITLE
Documentation for SolveType = 2

### ISSUE THREAD

**Issue by matbesancon** [JuliaOpt/Clp.jl#23]
From the README:
> SolveType - choose the solution method:
>     0 - dual simplex
>     1 - primal simplex
>     3 - barrier with crossover to optimal basis
>     4 - barrier without crossover to optimal basis
>     5 - automatic
What does option `SolveType=2` yield? This seems undocumented here.
By digging into Clp Doxygen: https://www.coin-or.org/Doxygen/Clp/classClpSolve.html#a3e2f401fca0ad2ea77c7486aa4d4c3f2  
2 corresponds to `usePrimalorSprint`, I have no idea what this means though, and no additional documentation for the term found.

**Comment by mbesancon** [comment]
Sprint / Sifting would be the same? Possible. I suppose this could be a documentation issue on Clp itself

**Comment by mlubin** [comment]
I would recommend asking on the Clp mailing list.

**Comment by mbesancon** [comment]
Answer from John Forrest from the CLP team:
> It is for reasonably homogeneous problems with many variables and does a series of primal solves on subsets of these variables.
Confirmation that it's the same idea as CLPLEX's sifting, from the CLP documentation:
```
Clp:sprint??
sprint(Crash) : Whether to try sprint crash
For long and thin problems this program may solve a series of small
problems created by taking a subset of the columns.  I introduced
the idea as 'Sprint' after an LP code of that name of the 60's which
tried the same tactic (not totally successfully).  Cplex calls it
'sifting'.  -1 is automatic choice, 0 is off, n is number of passes
<Range of values is -1 to 5000000;
    current -1>
```

**Comment by matbesancon** [comment]
This should be added to the doc as well

**Comment by odow** [comment]
Closed by #80
